{"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"authError","kind":"identifier"}],"platforms":["macOS"],"languages":["swift"]}]}],"identifier":{"url":"doc:\/\/Yorkie\/documentation\/Yorkie\/DocEventType\/authError","interfaceLanguage":"swift"},"schemaVersion":{"minor":3,"major":0,"patch":0},"hierarchy":{"paths":[["doc:\/\/Yorkie\/documentation\/Yorkie","doc:\/\/Yorkie\/documentation\/Yorkie\/DocEventType"]]},"metadata":{"externalID":"s:6Yorkie12DocEventTypeO9authErroryA2CmF","title":"DocEventType.authError","roleHeading":"Case","role":"symbol","fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"authError"}],"modules":[{"name":"Yorkie"}],"symbolKind":"case"},"abstract":[{"type":"codeVoice","code":"authError"},{"type":"text","text":" indicates an authorization failure in syncLoop or watchLoop."}],"sections":[],"kind":"symbol","variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/yorkie\/doceventtype\/autherror"]}],"references":{"doc://Yorkie/documentation/Yorkie/DocEventType":{"navigatorTitle":[{"kind":"identifier","text":"DocEventType"}],"identifier":"doc:\/\/Yorkie\/documentation\/Yorkie\/DocEventType","fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"kind":"identifier","text":"DocEventType"}],"kind":"symbol","type":"topic","title":"DocEventType","role":"symbol","abstract":[{"code":"DocEventType","type":"codeVoice"},{"type":"text","text":" is document event types"}],"url":"\/documentation\/yorkie\/doceventtype"},"doc://Yorkie/documentation/Yorkie/DocEventType/authError":{"role":"symbol","fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"authError"}],"title":"DocEventType.authError","kind":"symbol","identifier":"doc:\/\/Yorkie\/documentation\/Yorkie\/DocEventType\/authError","url":"\/documentation\/yorkie\/doceventtype\/autherror","type":"topic","abstract":[{"code":"authError","type":"codeVoice"},{"text":" indicates an authorization failure in syncLoop or watchLoop.","type":"text"}]},"doc://Yorkie/documentation/Yorkie":{"kind":"symbol","role":"collection","url":"\/documentation\/yorkie","title":"Yorkie","abstract":[],"type":"topic","identifier":"doc:\/\/Yorkie\/documentation\/Yorkie"}}}